Registry/Registry.EM_COMMON

```diff
@@ -693,6 +693,8 @@ state    real   ts_rainnc       ?!       misc      -         -      -        "TS
 # Time series of vertical profile of U, V, GHT, THETA and QVAPOR
 state    real   ts_u_profile   ?!k       misc      -         -      -        "TS_U_PROFILE"   "Wind u-Component, earth relative, vertical profile"
 state    real   ts_v_profile   ?!k       misc      -         -      -        "TS_V_PROFILE"   "Wind v-Component, earth relative, vertical profile"
+#PSH
```

Comment:
@davegill can comment authoritatively, but I think we're generally recommending that contributors not leave their initials around the code.

Reply:
@mgduda @davegill 
I'll remove them.